### ISSUE TITLE
Changing the 'Downloads' directory causes an error!

### ISSUE THREAD

**Issue by ProfessorCode212** [Unrud/video-downloader#82]
I changed the default location of the *Downloads* directory for VideoDownloader using `flatpak run --command=gsettings com.github.unrud.VideoDownloader set com.github.unrud.VideoDownloader download-folder '~/Videos/VideoDownloader'`. However, I an error when I click on *Open Download Location* and it doesn't appear in the specified directory. I have to get the video from the cache. I am on Linux Mint 19.3 Cinnamon.

![yt-dl_error](https://user-images.githubusercontent.com/78950242/109075389-0e261380-771f-11eb-83f2-2f1b8ae7d1c1.png)

![image](https://user-images.githubusercontent.com/78950242/109075417-17af7b80-771f-11eb-8ca4-84ab559251bf.png)

**Comment by GustavoPeredo** [comment]
I think this might be related to flatpak permissions, but I'm not entirely sure. Is it possible for you to run the following command and try again? 

`flatpak override com.github.unrud.VideoDownloader --filesystem=~/Videos`

(You can also change the directory permissions using Flatseal)

**Comment by ProfessorCode212** [comment]
> I think this might be related to flatpak permissions, but I'm not entirely sure. Is it possible for you to run the following command and try again?
> 
> `flatpak override com.github.unrud.VideoDownloader --filesystem=~/Videos`
> 
> (You can also change the directory permissions using Flatseal)

It didn't work. I still get the same error. Besides, Video Downloader has access to my home directory. There is no reason for it to not be able to access the _Videos_ folder.

![image](https://user-images.githubusercontent.com/78950242/109092515-b0082900-773c-11eb-9682-94a34a0ea0f5.png)

**Comment by GustavoPeredo** [comment]
Last thing before I start investigating on my local machine: do you have a folder named VideoDownloader under ~/Videos? If you don't, can you create one and try again?

**Comment by ProfessorCode212** [comment]
> Last thing before I start investigating on my local machine: do you have a folder named VideoDownloader under ~/Videos? If you don't, can you create one and try again?

I have a made a folder. It still doesn't work.

**Comment by ProfessorCode212** [comment]
> I think this might be related to flatpak permissions, but I'm not entirely sure. Is it possible for you to run the following command and try again?
> 
> `flatpak override com.github.unrud.VideoDownloader --filesystem=~/Videos`
> 
> (You can also change the directory permissions using Flatseal)

I got it to work. I re-ran the command to change the default _download_ directory for Video Downloader. This time, I used `xdg-videos/VideoDownloader` instead of `~/Videos/VideoDownloader`. I also gave VideoDownloader access to `xdg-videos` through Flatseal. Thanks for you help, @GustavoPeredo! I hope the documentation for changing the default directory gets updated, though.

![image](https://user-images.githubusercontent.com/78950242/109094471-032fab00-7740-11eb-9f4a-cb36b75e66f4.png)

![image](https://user-images.githubusercontent.com/78950242/109094211-9b796000-773f-11eb-87e1-252d8651add9.png)

https://user-images.githubusercontent.com/78950242/109094137-843a7280-773f-11eb-8bc2-9eac2159fa74.mp4
